<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>669</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>669</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>669</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Virtual Election Booth</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/lastLogo.png</normaloff>../images/lastLogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>550</width>
      <height>500</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(54, 54, 54);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>528</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>201</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 28pt &quot;Forte&quot;;
color:rgb(11,195,245);</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>201</width>
           <height>28</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>461</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>228</width>
           <height>38</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sign_ptn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 127);font-size:14pt;color:rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>Sign In </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>501</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="mail_lpl_2">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 0, 127);font:13pt;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>User Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_mail_2">
           <property name="maximumSize">
            <size>
             <width>351</width>
             <height>51</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);font:14pt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="pass_lpl_2">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>41</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>41</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 0, 127);font:13pt;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Password</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_pass_2">
           <property name="maximumSize">
            <size>
             <width>351</width>
             <height>51</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);font:14pt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="person_1">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 0, 0);font:20pt;
</string>
         </property>
         <property name="text">
          <string>person1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="person_2">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 0, 255);font:20pt;
</string>
         </property>
         <property name="text">
          <string>person2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>511</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>328</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="send_vote_2">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 255, 127);font-size:14pt;color:rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>550</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(54, 54, 54);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/5.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>550</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>550</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>550</width>
      <height>51</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Forte&quot;;
color:rgb(11,195,245);</string>
    </property>
    <property name="text">
     <string>Virtual Election Booth</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
